mmv1/products/logging/LogView.yaml

```diff
@@ -24,6 +24,7 @@ references: !ruby/object:Api::Resource::ReferenceLinks
   guides:
     'Official Documentation': 'https://cloud.google.com/logging/docs/apis'
   api: 'https://cloud.google.com/logging/docs/reference/v2/rest/v2/projects.locations.buckets.views'
+skip_sweeper: true
```

Comment:
Can you comment with the reasoning inline?

Reply:
Done